eclipse-scout-core/src/form/fields/groupbox/GroupBox.ts

```diff
@@ -283,6 +283,7 @@ export class GroupBox extends CompositeField implements GroupBoxModel {
       bodyLayoutConfig = new LogicalGridLayoutConfig();
     }
     this._setProperty('bodyLayoutConfig', LogicalGridLayoutConfig.ensure(bodyLayoutConfig));
+    LogicalGridLayoutConfig.initHtmlEnvChangeHandler(this, () => this.bodyLayoutConfig, layoutConfig => this.setBodyLayoutConfig(layoutConfig));
```

Comment:
Remark about the commit message of e7f880c32249eae5f56823b88ee13d84f0d07e88:

> Default values are now available directly on the layout config object.
> This makes it easier to access the actual values for custom calculations.

What exactly does this mean? Is there an example for such a "custom calculation"? And why are the default values needed to access the "actual values"? Are these the widget-specific values or the final, merged values?